src/component/feature/FeatureToggleList/FeatureToggleListTable/FeatureToggleListTable.tsx

```diff
@@ -0,0 +1,196 @@
+import { useEffect, useMemo, VFC } from 'react';
+import {
+    Link,
+    Table,
+    TableBody,
+    TableCell,
+    TableRow,
+    useMediaQuery,
+    useTheme,
+} from '@mui/material';
+import { Link as RouterLink } from 'react-router-dom';
+import { useGlobalFilter, useSortBy, useTable } from 'react-table';
+import useLoading from 'hooks/useLoading';
+import { SortableTableHeader } from 'component/common/Table/SortableTableHeader/SortableTableHeader';
+import { TableActions } from 'component/common/Table/TableActions/TableActions';
+import { TablePanel } from 'component/common/Table/TablePanel/TablePanel';
+import { TableToolbar } from 'component/common/Table/TableToolbar/TableToolbar';
+import TablePlaceholder from 'component/common/Table/TablePlaceholder/TablePlaceholder';
+import { SearchHighlightProvider } from 'component/common/Table/SearchHighlightContext/SearchHighlightContext';
+import { DateCell } from './DateCell/DateCell';
+import { FeatureNameCell } from './FeatureNameCell/FeatureNameCell';
+import { FeatureSeenCell } from './FeatureSeenCell/FeatureSeenCell';
+import { FeatureStaleCell } from './FeatureStaleCell/FeatureStaleCell';
+import { FeatureTypeCell } from './FeatureTypeCell/FeatureTypeCell';
+import { LinkCell } from './LinkCell/LinkCell';
+import { CreateFeatureButton } from '../../CreateFeatureButton/CreateFeatureButton';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+
+interface IExperimentProps {
+    data: Record<string, any>[];
```

Comment:
Would be great with some stricter types in this file, for the data and the columns.

Maybe we could use `v8.0.0-alpha.64` if that helps a lot.

Reply:
I hope it will be an easy upgrade. There is no up to date docs for v8. I'd like to wait until it is stable.